### PR TITLE
Registration page 

### PR DESCRIPTION
**Pull request by j-forcer90**
Final tests!!!!!!

### CODE REVIEW

**Review by jpat2398 — CHANGES_REQUESTED** [review]
remove tests from registration page

**Review by JTAfo1 — CHANGES_REQUESTED** [review]
delete registration

**Review by Zeeks17 — CHANGES_REQUESTED** [review]
delete tests on registration page

File: target/test-classes/Main.class
Comment (Zeeks17) [inline target/test-classes/Main.class:1]: clear target

**Review by Zeeks17 — CHANGES_REQUESTED** [review]

**Review by LayZ2001 — APPROVED** [review]
looks good

**Review by Zeeks17 — APPROVED** [review]

**Review by LayZ2001 — APPROVED** [review]

**Review by jpat2398 — APPROVED** [review]

**Review by JTAfo1 — APPROVED** [review]
good stuff

**Pull request merged**